mobile/src/components/interviews/common/maps/status-to-color.map.ts

```diff
@@ -0,0 +1,12 @@
+import { InterviewStatus } from '~/common/enums/enums';
+
+const statusToColor: Record<InterviewStatus, string> = {
+  [InterviewStatus.NEW]: '#ecbc02',
+  [InterviewStatus.REJECTED]: '#7e19f7',
+  [InterviewStatus.CANCELED]: '#f64848',
+  [InterviewStatus.COMPLETED]: '#7bdf00',
+  [InterviewStatus.PENDING]: '#03d7fc',
+  [InterviewStatus.IN_PROGRESS]: '#d605e8',
```

Comment:
Are you sure that we have the same colors in frontend part?

Reply:
This colors i was got from Vlada Kytaieva. 
She work on similar task on front-end

Reply:
I will double check